server/src/test/scala/org/scassandra/server/actors/ConnectionHandlerTest.scala

```diff
@@ -303,4 +320,9 @@ class ConnectionHandlerTest extends TestKit(ActorSystem("Test")) with Matchers w
     testActorRef ! Received(ByteString(startupMessage.toArray))
   }
 
+  private def clear(): Unit = {
```

Comment:
It is actually any messages not verified by previous tests. This is an issue when using the ImplicitSender trait. I asked in akka-dev and they recommend not using ImplicitSender but use a different TestProbe per test. I raised https://github.com/scassandra/scassandra-server/issues/82 so i don't forget to switch over all the tests.